src/ORM/Table.php

```diff
@@ -1436,6 +1435,36 @@ public function get($primaryKey, $options = [])
     }
 
     /**
+     * Handles the logic executing of a worker inside a transaction.
+     *
+     * @param bool $atomic Whether to execute the worker inside a database transaction.
+     * @param callable $worker The worker that will run inside the transaction.
+     * @return mixed
+     */
+    protected function _executeTransaction($atomic, callable $worker)
```

Comment:
Having the method signature as `_executeTransaction(callable $worker,  $atomic = true)` would be better in my opinion.